theme/less/_content.less

```diff
@@ -18,45 +18,47 @@ article p:first-of-type img
 
 article p
 {
-  margin: 0.5em 0 0 0;
+  margin: 14px 0 0 0;
 }
 
-article p + ul
+article ul
 {
-  margin-top: 0.3em;
+  margin: 20px 0 0 0;
```

Comment:
Isn't 20px too much? according to our discussion we need to decrease this value.

Reply:
Yes, previously the content text was pushing the footer, now after decreasing the footer size we don't have that issue. AFAIK that was the case we decided to decrease. Currently the size is seem to be consistent with the size `<p>` is pushing below `<ul>` elements in the content.

Reply:
I'll leave it like that to be consistent with Drupal version.